### ISSUE TITLE
Is this a typo with the extra `s`?

### ISSUE THREAD

**Issue by dragon788** [perpen/chromeos-runc#1]
https://github.com/perpen/chromeos-runc/blob/b49ad5c74843228198e91c1867dc1bbe8104b5c7/docker-daemon.sh#L6

I would love to reproduce some of your success on my Chromebook but there appear to be a couple hardcoded paths that don't exist on my system that may require some additional setup that isn't documented.

**Comment by perpen** [comment]
Yes, you have to create directories. I'm guessing rootfss is the plural of rootfs, sort of consistent with the chroots dir next to it.
This repo definitely wasn't meant to be usable out of the box, you have to know what you're doing.

But ignore docker-daemon.sh until you have a container running. I would suggest you focus on understanding runc and its config.
Steps would be:
1. Create the fs for your container: I used some version of https://github.com/moby/moby/blob/master/contrib/mkimage-arch.sh for this, which I may have run either from crouton or from a proper linux box, can't remember.  Copy it to /mnt/stateful_partition/chroots. 
2. From crosh: Manually edit the json config for runc to point to your fs, bind mount what's needed, tweak it until it works. At the beginning I must have been starting just a bash session in the container, to setup everything (in particular sshd), before switching on systemd. 
3. Once you get it working, look at how to automate the setup, and start the container on boot.
4. Optionally get docker working (this I'm not too confident about, lots of changes in docker).

But ofc in 2 years chrome os changed, the method may not be valid anymore. But I'd be surprised.
